dist/donut-chart/donut-chart.css

```diff
@@ -0,0 +1,66 @@
+.donut-chart {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  position: relative;
+}
+.donut-chart svg {
+  display: inline-block;
+  position: relative;
+  vertical-align: middle;
+}
+.donut-chart__segment {
+  opacity: 1;
+  transition: opacity 0.3s;
+}
+.donut-chart__segment.primary {
```

Comment:
Should be `.donut-chart__segment--primary` to be valid BEM.

If we feel that these segment fills might be used outside of donut charts, then we may want to promote them to their own block class with modifier, e.g. 'data-viz-segment data-viz-segment--primary`.

For now though lets go with the first suggestion and we can always refactor later to the second suggestion if we need to.